iothub/service/src/DeviceJobType.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.Azure.Devices
     [JsonConverter(typeof(StringEnumConverter))]
     public enum DeviceJobType
     {
+        /// <summary>
+        /// Unknown job type
```

Comment:
Add punctuation (dot at the end in this case) as these comments show up on Microsoft docs